sys/linux/socket_netlink_generic_devlink.txt

```diff
@@ -102,6 +104,18 @@ devlink_trap_set_policy {
 	DEVLINK_ATTR_TRAP_ACTION	nlattr[DEVLINK_ATTR_TRAP_ACTION, int8[DEVLINK_TRAP_ACTION_DROP:DEVLINK_TRAP_ACTION_TRAP]]
 } [packed, align[4]]
 
+devlink_trap_policer_get_policy {
+	dev_handle			devlink_handle
+	DEVLINK_ATTR_TRAP_POLICER_ID	nlattr[DEVLINK_ATTR_TRAP_POLICER_ID, int32[0:3]]
```

Comment:
`int32[1:3]`? Also it would be nice to add a comment that explains where these ids are coming from.

Reply:
The explanation is [added](https://github.com/google/syzkaller/pull/1893/files#diff-a917613db04eaaa13a6f7d577d6872aeR107-R109).

Reply:
OK, I see, 0 is a special value, thanks!